openmetadata-ui/src/main/resources/ui/src/components/AddWebhook/AddWebhook.tsx

```diff
@@ -730,7 +732,7 @@ const AddWebhook: FunctionComponent<AddWebhookProps> = ({
                       <input
                         readOnly
                         className="tw-form-inputs tw-px-3 tw-py-1"
-                        data-testid="connection-timeout"
+                        data-testid="secret-key"
```

Comment:
This might be a mistake

Reply:
Hey, i checked on this part, we have to change to "secret-key", as this feild belog to "secret key", let me know if need more details on this.

Reply:
Please change the name and id attributes as well

Reply:
Code updated